### DISCUSSION TITLE
打造优雅高效的写作环境 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#139] [Announcements]
# 打造优雅高效的写作环境 · BMPI

本文介绍了使用GitHub Pages以及Hexo工具打造个人优化高效的写作系统

https://www.bmpi.dev/self/build-write-tool-v1/

**Comment by houdq** [comment]
好喜欢UP主的 theme 不知道会不会开源啊👍

**Reply by madawei2699** [reply]
一直是开源的啊，去我的 GitHub 看看就能找到

**Comment by fantingsheng** [comment]
不知道怎么可以和博主交换友链 https://spacexcode.com
